lib/pymedphys/tests/dicom/test_connect.py

```diff
@@ -295,3 +297,131 @@ def test_dicom_listener_cli(test_dataset):
         file_path = _build_hierarchical_path_to_plan(test_directory, test_dataset)
         read_dataset = pydicom.read_file(file_path)
         assert read_dataset.SeriesInstanceUID == test_dataset.SeriesInstanceUID
+
+
+@pytest.mark.pydicom
+def test_dicom_sender(test_dataset):
+    """Test sending DICOM objects using the DicomSender"""
+
+    scp_ae_title = "PYMEDPHYSTEST"
+
+    with tempfile.TemporaryDirectory() as tmp_directory:
+
+        test_directory = pathlib.Path(tmp_directory)
+        receive_directory = test_directory.joinpath("receive")
+        receive_directory.mkdir()
+
+        listener_command = [
+            pmp_test_utils.get_executable_even_when_embedded(),
+            "-m",
+            "pymedphys",
+            "dicom",
+            "listen",
+            str(TEST_PORT),
+            "-d",
+            str(receive_directory),
+            "-a",
+            str(scp_ae_title),
+        ]
+
+        with process(listener_command):
+            time.sleep(1)
+            dicom_sender = DicomSender(
```

Comment:
@sjswerdloff what are your thoughts on naming here?

Given DICOM is an acronym should this be named `DICOMSender`?

Reply:
@SimonBiggs @sjswerdloff Any preference here before I merge this PR?

Reply:
Let's keep it as you have it @pchlap. And let's apply the upper camel case version of the following rules throughout the codebase in cases like these in the future:

https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined

Reply:
That style guide linked above agrees with your naming approach.

Reply:
Talk about bike shedding. Sorry about this @pchlap, when looking at pep-8 it says the following:

> When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError

So, let's follow pep-8 when there is ambiguity. So... DICOMSender it is... (If that's okay with you @pchlap

Reply:
You know what... I had a bit more of a read... Let's go with that Google style guide above and not use PEP-8 for this case. See https://stackoverflow.com/a/2853609 for more musings.

So... Just to bounce back and forth one last time, let's stick with `DicomSender`. And as time goes on, let's aim to make the codebase align with that approach to acronyms for reasons of consistency.

Reply:
... moral of the story ... I can't even boat-shed effectively when sick ...

Reply:
Thanks @SimonBiggs, ok sounds good. Will leave it as is. I will merge this PR and will follow up soon (hopefully later this week) with a DicomQueryRetriever class :)